articles/9-tips-for-student.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "9 tips I wish I knew when I was a student"
+tags: ["community", "school", "orientation", "tech", "GeeksBlaBla"]
+keywords: ["GeeksBlabla", "school", "orientation"]
+pubDatetime: 2024-12-01
+authors: ["iduoad"]
+slug: tips-for-students
+description: "Discover simple and practical advice to help students navigate school life, from learning effectively to standing out, and make the most of your time as a tech student."
+---
+
+There’s a saying I used to like: "School is like a free trial for work/life." This is somewhat true because school is the phase in life where we can make mistakes and learn the most.
+
+On the other hand, school can sometimes be overwhelming, especially for tech students. This can be due to peer pressure, constant comparisons, limited mentoring, scarce resources, and the lack of guidance on what to do, and what not to do.
+
+Since it’s PFE (end-of-study project) internship season, I thought of sharing some school tips and tricks that worked for me, as well as lessons I learned either during my student years or later on after I graduated.
+
+## 1. Learn as much as you can
+
+With all the GPTs and Claudes out there, reading a book or attending a class might feel like a thing of the past. If an AI chatbot can answer all your final exam questions, why even bother learning, right?
+
+Here’s the thing: in this age of easily accessible information, and Gen AI, learning is no longer just about acquiring knowledge. It’s about the cognitive exercise.
+
+When you consistently learn (even if the topics are irrelevant) your brain gets used to absorbing and processing information. You become a better learner in the process. This means that when you’re faced with something new, you’ll pick it up faster and more efficiently.
+
+Remember the free trial analogy? Once that trial ends (aka you graduate), the expectations get higher, and the time to meet them gets shorter. If you’ve built the habit of being a good learner, you’ll have a better chance at navigating your career with ease.
+
+All in all, keep your brain active and working. The best way to do that? Learn.
+
+> 💡 Tip: Mix slow learning experiences, like reading books or taking courses, with faster ones, like watching YouTube tutorials or reading blog posts. This will train your brain to adapt to different modes of learning and help you create your own style.
+
+#### Useful resources
+🔗 [Awesome Moroccan podcasts](https://awesome-morocco.dev/podcasts): List of Moroccan tech podcasts.
+🔗 [The Missing Semester](https://www.missingsemester.io/): Courses about useful topics often not thaught in school.
+🔗 [MIT Courseware](https://ocw.mit.edu/): MIT course about various tech fields.
+🔗 [Roadmap.sh](https://roadmap.sh/): Roadmap for Software Engineers with resources and projects.
+
+## 2. Keep Moving
+
```

Comment:
Would love to add here, if you didn't find a community or club  in your school you can start one

Reply:
Done! Sorry for te3tila